openlibrary/plugins/openlibrary/js/cover_add_manage.js

```diff
@@ -0,0 +1,58 @@
+export default function(){
+    // page may not be loaded via iframe
+    if (parent && parent.closeThrobber) {
+        parent.closeThrobber();
+    }
+    $('#form.addcover-form').submit(function(event) {
+        var file;
+        var url;
+        var coverid;
```

Comment:
any reason these are declared here but set below? All of them look static to me

Reply:
Lint said that variables needed to be declared at the top of a function, though your suggestion to move the two functions they use to the top of the file allowed me to declare and instantiate the variables in one line.